.github/workflows/build_container_from_ref.yml

```diff
@@ -0,0 +1,44 @@
+name: "Build container from arbitrary repo / ref"
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'The receptor repository to build from.'
+        required: true
+        default: 'ansible/receptor'
+      ref:
+        description: 'The ref to build. Can be a branch or any other valid ref.'
+        required: true
+        default: 'devel'
+      tag:
+        description: 'The tag for the container.'
+        required: true
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.ref }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Install build dependencies
+        run: |
+          pip install build
+
+      - name: Build Multiarch Image & Push To Quay
+        run: |
+          make container REPO=quay.io/aap-ci/receptor TAG=$TAG
```

Comment:
I did not want to populate `ansible/receptor` with random tags, but this one does not seem the better namespace either. Thoughts?

Reply:
moved to ghcr.io